Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -75,6 +75,8 @@ enum ConfigureStrings {
 
     case sk2_required_for_swiftui_paywalls
 
+    case could_not_defer_store_messages(errorMessage: String)
```

Comment:
To avoid passing a random string here, we typically make these take an Error and format it inside:
```suggestion
    case could_not_defer_store_messages(Error)
```